.jshintrc

```diff
@@ -2,6 +2,5 @@
   "esversion": 6,
   "node": true,
   "undef": true,
-  "unused": true,
-  "validthis": true
+  "unused": true
```

Comment:
Nice

Reply:
I had to temporarily add `"validthis": true` to allow the "as-is" source to pass, but did so knowing that I could immediately remove it in the follow up changeset.

Reply:
That is exactly the nuanced procedure that I was complimenting :)